.github/workflows/main.yml

```diff
@@ -302,7 +302,7 @@ jobs:
     - run: $CENTOS cargo build --release --bin wasmtime --bin wasm2obj
       shell: bash
     # Build `libwasmtime.so`
-    - run: $CENTOS cargo build --release --manifest-path crates/api/Cargo.toml
+    - run: $CENTOS cargo build --release --manifest-path crates/capi/Cargo.toml
```

Comment:
Could we call it `wasmtime-c-api` instead of `wasmtime-capi`? That makes it reflect the `wasm-c-api` interface it's implementing. And I'd find `wasmtime-c-api` easier to read than `wasmtime-capi`, since my eyes default to reading "capi" as if it were pronounced "cappy".